### PR TITLE
Better help text for activation limit

### PR DESCRIPTION
**Pull request by Himavanth**
## Description
Fixes https://github.com/adobe/aio-cli-plugin-runtime/issues/229

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by Himavanth** [comment]
@meryllblanchet Could you check the text change ?

File: src/commands/runtime/activation/list.js [hunk 1]
```diff
@@ -214,7 +214,7 @@ ActivationList.flags = {
   // example usage:  aio runtime:activation:list --limit 10 --skip 2
   limit: flags.integer({
     char: 'l',
-    description: `only return LIMIT number of activations from the collection with a maximum LIMIT of ${ActivationList.limits.max} activations (default 30)`
+    description: `only return LIMIT number of activations (maximum ${ActivationList.limits.max}). Note that the maximum could be set to a lesser value by the server`
```
Comment (meryllblanchet) [inline src/commands/runtime/activation/list.js:217]: What about `Note that the maximum set for I/O Runtime is 50` ?

**Review by meryllblanchet — CHANGES_REQUESTED** [review]

**Review by meryllblanchet — APPROVED** [review]

File: src/commands/runtime/activation/list.js [hunk 2]
```diff
@@ -214,7 +214,7 @@ ActivationList.flags = {
   // example usage:  aio runtime:activation:list --limit 10 --skip 2
   limit: flags.integer({
     char: 'l',
-    description: `only return LIMIT number of activations from the collection with a maximum LIMIT of ${ActivationList.limits.max} activations (default 30)`
+    description: `only return LIMIT number of activations (maximum ${ActivationList.limits.max}). Note that the maximum set for Adobe I/O Runtime is 50`
```
Comment (rabbah) [inline src/commands/runtime/activation/list.js:217]: Do you have to repeat this message on al the list commands?

mid the Max says 50 already, do you need the second Adobe part :)

**Comment by Himavanth** [comment]
On second thought, since these limits are set by the server and there is no way for the CLI to know the limits in advance, I think the CLI should refrain from saying anything about the limits and just pass it on. 
An invalid limit will return an error message from the server with the valid range. eg:- `the value '51' is not in the range of 0 to 50 for actions`.
This would also avoid us having to change the limits in future if the server changes them.

**Comment by meryllblanchet** [comment]
Makes sense, what description would you then propose for the CLI listing commands?

**Comment by Himavanth** [comment]
I would propose to keep the existing text after removing the absolute numbers for the limits.
So the description would become `only return LIMIT number of activations` instead of `only return LIMIT number of activations from the collection with a maximum LIMIT of 50 activations (default 30)`

**Pull request merged**